### ISSUE TITLE
New Block Creation Requests

### ISSUE THREAD

**Issue by kgracey** [parallaxinc/solo#693]
**These are from customers:**
Soil Moisture Sensor (Parallax #28092)
UV Light Sensor (Parallax #28091)
HX711 Load Cell Amplifier (SparkFun #13879)

**From Rob Faludi, of Digi, for our upcoming cooperative contest:**
XBee Cellular Modem

**Comment by zfi** [comment]
@kgracey, it looks like we could use the A/D block if the project is running on a Propeller Activity Board with some minor adjustments for the 28091 and 28092. If we're looking to support additional devices, then we will build new blocks.

**Comment by kgracey** [comment]
Agreed, @zfi I think the A/D-requiring parts should only accompany the Propeller Activity Board type.
